docs/swagger.yaml

```diff
@@ -71,16 +71,45 @@ paths:
           name: Accept
           required: true
           description: Requested image format
-        - schema:
-            type: string
-          in: header
-          name: Ocp-Apim-Subscription-Key
-          description: Subscription Key
-          required: true
+        - $ref: '#/components/parameters/Ocp-Apim-Subscription-Key'
       tags:
         - QR
       security:
         - BearerToken: []
+  /v1/merchantRedirect:
+    post:
+      summary: CreateMerchantRedirectQr
+      operationId: CreateMerchantRedirectQr
+      responses:
+        '201':
```

Comment:
Maybe we should use http 200 here, to keep in line with the other endpoint. We should also add the 400, 401 and 415 responses thats present on the other endpoint

Reply:
agreed, should we add the 409 as well? for conflicts in the db?

Reply:
Yes! Very good idea